examples/notebooks/FastaiLRFinder_MNIST.ipynb

```diff
@@ -238,7 +238,7 @@
     "id": "NcT19wqkmvd6"
```

Comment:
Let's update this notebook as we discussed previously:
- Let's split this cell into multiple ones:
```
trainer = create_supervised_trainer(model, optimizer, criterion, device=device)
ProgressBar(persist=True).attach(trainer, output_transform=lambda x: {"batch loss": x})

lr_finder = FastaiLRFinder()
to_save={'model': model, 'optimizer': optimizer}
with lr_finder.attach(trainer, to_save, diverge_th=1.5) as trainer_with_lr_finder:
    trainer_with_lr_finder.run(trainloader)
    
trainer.run(trainloader, max_epochs=10)

evaluator = create_supervised_evaluator(model, metrics={"acc": Accuracy(), "loss": Loss(nn.NLLLoss())}, device=device)
evaluator.run(testloader)

print(evaluator.state.metrics)
```
->
```python
# CELL 1
trainer = create_supervised_trainer(model, optimizer, criterion, device=device)
ProgressBar(persist=True).attach(trainer, output_transform=lambda x: {"batch loss": x})

lr_finder = FastaiLRFinder()
to_save={'model': model, 'optimizer': optimizer}
with lr_finder.attach(trainer, to_save, diverge_th=1.5) as trainer_with_lr_finder:
    trainer_with_lr_finder.run(trainloader)
    
# CELL 2 - Text
Let's plot the found optimal learning rate.

# CELL 3
from matplotlib import pyplot as plt
ax = lr_finder.plot()
plt.axes(ax)
plt.show()

print("Suggested LR", lr_finder.lr_suggestion())

# CELL 4 - Text
Let's compare training results: a) using a fixed learning rate (`lr=3e-4`) and b) with applying optimal learning rate.

# CELL 5
trainer.run(trainloader, max_epochs=10)

evaluator = create_supervised_evaluator(model, metrics={"acc": Accuracy(), "loss": Loss(nn.NLLLoss())}, device=device)
evaluator.run(testloader)

print(evaluator.state.metrics)
```